components/RoastDetailCardProfile/index.js

```diff
@@ -0,0 +1,106 @@
+import defaultPic from "@/public/default.jpg";
+import styled from "styled-components";
+import { useState } from "react";
+import {
+  StyDiv,
+  StyImage,
+  StyDivText,
+  StyHTwo,
+  StyP,
+  StyRating,
+  StyNumberRating,
+  StyButtonDiv,
+  StyButton,
+} from "../RoastDetailCard";
+import { StyLabel } from "../Form";
+
+export default function RoastDetailCardProfile({
+  name,
+  roaster,
+  arabica,
+  robusta,
+  level,
+  provenance,
+  score,
+  edit,
+  setEdit,
+  rateEdit,
+  setRateEdit,
+  submitRating,
+}) {
+  const [rating, setRating] = useState(50);
+
+  function showRating(event) {
+    let value = event.target.value;
```

Comment:
I did this because with the arabica/robusta slider, I had the problem with getting a live value due to some math I implemented in the function. Here it is indeed not needed.